src/page-multi-facility/FacilityInputForm.tsx

```diff
@@ -213,7 +242,25 @@ const FacilityInputForm: React.FC<Props> = ({ scenarioId }) => {
         <div className="mt-5 mb-5 border-b border-gray-300" />
 
         <RtChartContainer>
-          <RtTimeseries data={rtTimeseriesData} />
+          <ChartHeader>
+            <ChartTitle>Rate of Spread</ChartTitle>
+            {rtTimeseriesData && (
+              <HelpButtonWithTooltip>
+                This chart shows the rate of spread of Covid-19 over time. When
+                the Rt value is above 1 (the red line), the virus is spreading.
+                If the Rt value is below 1, the virus is on track to be
+                extinguished at this facility.
+              </HelpButtonWithTooltip>
+            )}
+          </ChartHeader>
+          {rtTimeseriesData ? (
+            <RtTimeseries data={rtTimeseriesData} />
+          ) : (
+            <RtChartEmptyState>
+              Live rate of spread could not be calculated for this facility.
+              Click here to add at least 10 days of confirmed case data.
+            </RtChartEmptyState>
+          )}
```

Comment:
I might put all this in the `RtTimeseriesContainer` component instead, to make it more self-contained and portable. it's already deciding whether to show a loading state or to suppress the chart so this would be a natural extension of that I think

Reply:
@macfarlandian 👍 I get what you mean, will give that a try

Reply:
@macfarlandian sorry for the delay, got caught up with some non-Recidiviz stuff. 

I can't figure out what you mean. Would you be up for pairing tomorrow if you're free? Am free all day.

//

The other question I have is: as-is, there are some instances where you get into this state (started out with just one day data, then add one day, not enough for the full Rt chart), that I'm not sure how to get past. Maybe that would be solved if this red empty state box is moved here https://github.com/Recidiviz/covid19-dashboard/blob/master/src/rt-timeseries/RtTimeseriesContainer.tsx#L17 but got stuck.

![image](https://user-images.githubusercontent.com/1372946/82277327-6b916680-993c-11ea-9ab3-62c6e4595466.png)